Makefile

```diff
@@ -58,3 +60,13 @@ test-unit:
 .PHONY: test-e2e
 test-e2e: binaries
 	ginkgo test/e2e
+
+.PHONY: deploy
+deploy:
+	$(KUBECTL) create -f $(TOPOLOGYAPI_MANIFESTS)/crd.yaml
```

Comment:
Good idea to refer to the CRD manifest in the noderesourcetopology-api!

Reply:
:)